### PR TITLE
Acceptor server integration into RG University.

### PR DESCRIPTION
**Pull request by Shico-UA**
@z4y4ts 
1. Commit: https://github.com/raccoongang/edx-platform/pull/80/commits/8575ca3c16035eee5d3ae88e38a50c60ddc52501
2. Commit: https://github.com/raccoongang/edx-platform/pull/80/commits/6c6815c0ccb4429eeae9529e1d6bf9df986c1b4f

### CODE REVIEW

File: lms/envs/common.py [hunk 1]
```diff
@@ -30,9 +30,11 @@
 # pylint: disable=invalid-name
 
 import imp
+import random
 import sys
 import os
 
+from celery.schedules import crontab
```
Comment (z4y4ts) [inline lms/envs/common.py]: Why do you add imports here, but not in `lms/envs/aws.py`?

Comment (Shico-UA) [inline lms/envs/common.py]: I agree, aws.py more specific to our celery settings. Refactored.

File: lms/envs/aws.py [hunk 2]
```diff
@@ -18,9 +18,11 @@
 
 import datetime
 import json
+import random
 import warnings
 
 import dateutil
+from celery.schedules import crontab
```
Comment (DmytryiStriletskyi) [inline lms/envs/aws.py]: Fix import order.

File: openedx/core/djangoapps/edx_global_analytics/utils.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
+"""
+Helpers for the edX global analytics application.
+"""
+
+import calendar
+import datetime
+
+
```
Comment (DmytryiStriletskyi) [inline openedx/core/djangoapps/edx_global_analytics/utils.py]: Remove one excess empty line between imports.

File: openedx/core/djangoapps/edx_global_analytics/utils.py [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    if cache_timeout is not None:
+        return cache_instance_data(name_to_cache, statistics_queries[query_type], cache_timeout)
+
+    return statistics_queries[query_type]
+
+
+def cache_instance_data(name_to_cache, query_result, cache_timeout):
+    """
+     Caches queries, that calculate particular instance data,
```
Comment (DmytryiStriletskyi) [inline openedx/core/djangoapps/edx_global_analytics/utils.py]: Delete excess whitespace character in docstring.

File: openedx/core/djangoapps/edx_global_analytics/utils.py (hunk 3 above)
Comment (Shico-UA) [inline openedx/core/djangoapps/edx_global_analytics/utils.py]: Done

File: openedx/core/djangoapps/edx_global_analytics/utils.py (hunk 4 above)
Comment (Shico-UA) [inline openedx/core/djangoapps/edx_global_analytics/utils.py]: Done

**Pull request merged**